TracksDemo/TracksDemo/CrashLoggingRootViewController.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+import AutomatticTracks
+
+class CrashLoggingRootViewController: UIHostingController<CrashLoggingView> {
+
+    let crashLogging = CrashLogging(dataProvider: CrashLoggingDataSource())
+
+    init() {
+        crashLogging.start()
```

Comment:
I had to update this to a brute for the iOS demo to build.

```suggestion
        _ = try! crashLogging.start()
```

Reply:
Shoot, sorry about that – fixed in b3ba4d2c5358d4ff91524025db5026d21d37f695 (in a slightly different way)

Reply:
Slightly different but still effective 👍